Sources/Display/Platform/Win32/win32_window.cpp

```diff
@@ -1685,8 +1735,6 @@ void Win32Window::create_hid_devices()
 
 void Win32Window::get_styles_from_description(const DisplayWindowDescription &desc, DWORD &style, DWORD &ex_style)
 {
-	style = 0;
-	ex_style = 0;
```

Comment:
Because caller could initialize arguments with his own styles already and this lines will override predefined styles. E.g. I always init window style with WS_OVERLAPPEDWINDOW|WS_VISIBLE, and these lines will override all predefined styles. I thought the purpose of this function was to add style bits, not to override style completely. Correct me if I'm wrong.